src/Compilers/Server/VBCSCompiler/AnalyzerConsistencyChecker.cs

```diff
@@ -15,19 +15,12 @@ namespace Microsoft.CodeAnalysis.CompilerServer
 {
     internal static class AnalyzerConsistencyChecker
     {
-        private static readonly ImmutableArray<string> s_defaultIgnorableReferenceNames = ImmutableArray.Create("mscorlib", "System", "Microsoft.CodeAnalysis", "netstandard");
-
         public static bool Check(string baseDirectory, IEnumerable<CommandLineAnalyzerReference> analyzerReferences, IAnalyzerAssemblyLoader loader, IEnumerable<string>? ignorableReferenceNames = null)
```

Comment:
>ignorableReferenceNames [](start = 170, length = 23)

Is this parameter still used?

Reply:
No. Thought I had deleted that. Will fix.